internal/controller/drplacementcontrol_controller.go

```diff
@@ -1531,8 +1535,8 @@ func (r *DRPlacementControlReconciler) getClusterDecisionFromPlacementRule(plRul
 // getPlacementDecisionFromPlacement returns a PlacementDecision for the passed in Placement if found, and nil otherwise
 // - The PlacementDecision is determined by listing all PlacementDecisions in the Placement namespace filtered on the
 // Placement label as set by OCM
-// - Function also ensures there is only one decision for a Placement, as the needed by the Ramen orchestrators, and
-// if not returns an error
+// - Function also ensures there is only one non-failover retained decision for a Placement, as the needed by the
+// Ramen orchestrators, and if not returns an error
```

Comment:
Should we add another reason for the non-retaied case (e.g. "Current")? It will make the code and the status more clear, and we will have a better name to refer to this. In e2e will not use this reason since we must support older version without a reason.